data/libs/Vector.lua

```diff
@@ -6,9 +6,9 @@ local Vector = {}
 local function binaryOp(a, b, op)
 	local typ = type(b)
 	if(typ == "number") then
-		return Vector(a.x + b, a.y + b, a.z + b)
```

Comment:
I personally would have split this into its own commit, but this is a really subjective matter :)